gradle/libs.versions.toml

```diff
@@ -3,7 +3,7 @@ agp = "8.0.1"
 androidTools = "31.0.1" # == 23.0.0 + agp version
 bytebuddy = "1.14.4"
 composeCompiler = "1.4.7"
-javaTarget = "11"
+javaTarget = "17"
```

Comment:
...or maybe this is the thing I'm thinking of...

i.e., building on 17 is fine, but should remain targeting 11?